src/main/java/org/myrobotlab/service/Runtime.java

```diff
@@ -2697,6 +2698,12 @@ public static Double getBatteryLevel() {
   @Override
   public void setLocale(String code) {
     locale = new Locale(code);
+    // attempt to reload all services localizations

```

Comment:
This will override the localization for every service..  if you call setLocale at the runtime level,  then you'll have to go back through and reset the locale on speech recognition and on speech synthesis to ensure that those services are still operating in the proper language.

Reply:
uses Service.setLocale .. no longer an issue